opencti-platform/opencti-front/src/utils/Charts.js

```diff
@@ -35,8 +35,38 @@ export const lineChartOptions = (
   chart: {
     type: 'line',
     background: 'transparent',
+    events: {
+      mouseMove(e) {
+        e.fromElement.style.cursor = 'pointer';
+      },
+    },
     toolbar: {
-      show: false,
+      show: true,
+      tools: {
+        download: true,
+        zoom: false,
+        zoomin: false,
+        zoomout: false,
+        pan: false,
+        reset: false,
+      },
+      export: {
+        csv: {
+          filename: undefined,
```

Comment:
what about giving a default file name instead of 'undefined'? (the name of the graph)

Reply:
We give a 'undefined' name because every charts is used severals times in differents components in the platform. So we can't really give the name of the file, as we don't know which one is it.
But, ApexCharts provide the ID of the charts as name of the file.